fe/fe-core/pom.xml

```diff
@@ -723,6 +723,11 @@ under the License.
             <artifactId>trino-parser</artifactId>
             <version>385</version>
         </dependency>
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
```

Comment:
Why do we use this jar, we already have `jackson` in it. And I found that `json-simple` last update was Mar 21, 2012, it looks like it was never maintained.